.github/workflows/ci.yaml

```diff
@@ -190,9 +190,6 @@ jobs:
         run: |
           git remote -v
           git fetch --depth=1 origin master
-      - name: Only allow go.mod and go.sum changes
-        run: |
-          find . -type f ! -name 'go.mod' ! -name 'go.sum' -exec git diff --exit-code origin/master -- {} +
```

Comment:
Question... why are we trusting NSMBot to do more than update go.mod and go.sum ?

Reply:
OK... I see... because updates from cmd-template... could we please still scope this down for security reasons?

I'm OK with letting NSMBot have a little broader scope... but am a bit worried about having *no* limits to its powers :)

Reply:
> OK... I see... because updates from cmd-template... could we please still scope this down for security reasons?

Sure! https://github.com/networkservicemesh/cmd-template/pull/5